local/protocol/locate.go

```diff
@@ -0,0 +1,34 @@
+package protocol
+
+import (
+	"fmt"
+	"os"
+	"path"
+)
+
+// Utilities to deal with Scoot instances.
```

Comment:
It's a little weird that this lives in the same place as the grpc protocol. What's the reasoning here?